testing/skopeo/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Ed Robinson <ed@reevoo.com>
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=skopeo
+pkgver=0.1.23
+pkgrel=0
+pkgdesc="Work with remote images registries - retrieving information, images, signing content"
+url="https://github.com/projectatomic/skopeo"
+arch="x86_64"
+license="ASL 2.0"
+makedepends="go btrfs-progs-dev gnupg gpgme-dev ostree-dev lvm2-dev bash"
+options="!strip"
```

Comment:
We used to not strip go binaries because of some FUD, but then we realised that they can be normally striped as any other binary. However, it remained unchanged in existing abuilds. If you're not sure that it doesn't work stripped, then please remove `!strip`.